<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="24" hostname="LAPTOP-2KQV9F30" failures="1" tests="1" name="Default test" time="36.909" errors="0" timestamp="2025-05-21T20:16:17 CDT">
  <testcase classname="tests.ArrayTest" name="ValidcodeandSubmitSortedSquares" time="29.884">
    <failure type="java.lang.RuntimeException" message="Sheet not found: PracticeQuestion">
      <![CDATA[java.lang.RuntimeException: Sheet not found: PracticeQuestion
at utilities.ExcelReader.getDataByScenario(ExcelReader.java:32)
at tests.ArrayTest.ValidcodeandSubmitSortedSquares(ArrayTest.java:343)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- ValidcodeandSubmitSortedSquares -->
  <testcase classname="tests.ArrayTest" name="ValidcodeandRunSearchArray" time="0.0">
    <ignored/>
  </testcase> <!-- ValidcodeandRunSearchArray -->
  <testcase classname="tests.ArrayTest" name="inValidcodeandRunMAX" time="0.0">
    <ignored/>
  </testcase> <!-- inValidcodeandRunMAX -->
  <testcase classname="tests.ArrayTest" name="inValidcodeandSubmitMAX" time="0.0">
    <ignored/>
  </testcase> <!-- inValidcodeandSubmitMAX -->
  <testcase classname="tests.ArrayTest" name="Signout" time="0.0">
    <ignored/>
  </testcase> <!-- Signout -->
  <testcase classname="tests.ArrayTest" name="inValidcodeandSubmitSortedSquares" time="0.0">
    <ignored/>
  </testcase> <!-- inValidcodeandSubmitSortedSquares -->
  <testcase classname="tests.ArrayTest" name="validPythonCodeforTryEditor" time="0.0">
    <ignored/>
  </testcase> <!-- validPythonCodeforTryEditor -->
  <testcase classname="tests.ArrayTest" name="verifyArraysNavigation" time="0.0">
    <ignored/>
  </testcase> <!-- verifyArraysNavigation -->
  <testcase classname="tests.ArrayTest" name="ValidcodeandSubmitNumber" time="0.0">
    <ignored/>
  </testcase> <!-- ValidcodeandSubmitNumber -->
  <testcase classname="tests.ArrayTest" name="ValidcodeandSubmitMAX" time="0.0">
    <ignored/>
  </testcase> <!-- ValidcodeandSubmitMAX -->
  <testcase classname="tests.ArrayTest" name="inValidcodeandSubmitNumber" time="0.0">
    <ignored/>
  </testcase> <!-- inValidcodeandSubmitNumber -->
  <testcase classname="tests.ArrayTest" name="inValidcodeandSubmitSearchArray" time="0.0">
    <ignored/>
  </testcase> <!-- inValidcodeandSubmitSearchArray -->
  <testcase classname="tests.ArrayTest" name="ValidcodeandRunNumber" time="0.0">
    <ignored/>
  </testcase> <!-- ValidcodeandRunNumber -->
  <testcase classname="tests.ArrayTest" name="inValidcodeandRunSearchArray" time="0.0">
    <ignored/>
  </testcase> <!-- inValidcodeandRunSearchArray -->
  <testcase classname="tests.ArrayTest" name="InvalidPythonCodeforTryEditor" time="0.0">
    <ignored/>
  </testcase> <!-- InvalidPythonCodeforTryEditor -->
  <testcase classname="tests.ArrayTest" name="verifyPracticeQuestionNavigation" time="0.0">
    <ignored/>
  </testcase> <!-- verifyPracticeQuestionNavigation -->
  <testcase classname="tests.ArrayTest" name="verifyUserIsInArrayPage" time="0.0">
    <ignored/>
  </testcase> <!-- verifyUserIsInArrayPage -->
  <testcase classname="tests.ArrayTest" name="ValidcodeandRunSortedSquares" time="0.0">
    <ignored/>
  </testcase> <!-- ValidcodeandRunSortedSquares -->
  <testcase classname="tests.ArrayTest" name="ValidcodeandRunMAX" time="0.0">
    <ignored/>
  </testcase> <!-- ValidcodeandRunMAX -->
  <testcase classname="tests.ArrayTest" name="inValidcodeandRunSortedSquares" time="0.0">
    <ignored/>
  </testcase> <!-- inValidcodeandRunSortedSquares -->
  <testcase classname="tests.ArrayTest" name="inValidcodeandRunNumber" time="0.0">
    <ignored/>
  </testcase> <!-- inValidcodeandRunNumber -->
  <testcase classname="tests.ArrayTest" name="VerifyTryHere" time="0.0">
    <ignored/>
  </testcase> <!-- VerifyTryHere -->
  <testcase classname="tests.ArrayTest" name="verifyRunButtonWithoutEnteringCode" time="0.0">
    <ignored/>
  </testcase> <!-- verifyRunButtonWithoutEnteringCode -->
  <testcase classname="tests.ArrayTest" name="ValidcodeandSubmitSearchArray" time="0.0">
    <ignored/>
  </testcase> <!-- ValidcodeandSubmitSearchArray -->
  <testcase classname="tests.ArrayTest" name="VerifyPracticeQuestion" time="0.0">
    <ignored/>
  </testcase> <!-- VerifyPracticeQuestion -->
</testsuite> <!-- Default test -->
